test/benchmark/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+---
+version: '3.4'
+services:
+  weaviate:
+    command:
+    - --host
+    - 0.0.0.0
+    - --port
+    - '8080'
+    - --scheme
+    - http
+    image: semitechnologies/weaviate:latest
```

Comment:
shouldn't we always specify here explicitly the version that we want to test?

Reply:
As discussed I changed the benchmark script to build a docker image from the current source and then start that using the tools/test/run_ci_server script. This file has been removed :)